docs/source/software/systemwide/matlab.md

```diff
@@ -62,94 +62,379 @@ Use the following procedure if you need to install a MATLAB toolbox:
+
+Setting an email to receive updates on the SLURM job:
+
+```{code-block} matlab
+>> % Request email notification of job status
+>> c.AdditionalProperties.EmailAddress = 'user-id@northeastern.edu';
+```
+
+Setting the number of GPUs used and the type of [GPU](https://rc-docs.northeastern.edu/en/latest/hardware/hardware_overview.html#gpu-hardware):
 
-This section provides instructions for submitting a job to the cluster for scaling calculations on an integer factorization sample problem. The complexity of this problem increases with the magnitude of the number, making it computationally intensive. To perform these calculations, we will use the `myParallelAlgorithmFcn.m` MATLAB function. Please note that this section assumes you have already started a MATLAB Parallel Server.
+```{code-block} matlab
+>> % Specify number of GPUs (default: 0)
+>> c.AdditionalProperties.GPUsPerNode = 1;
+>> c.AdditionalProperties.GPUCard = 'gpu-card';
+```
+
+```{note}
+If you need a GPU, you need to make sure the partition you are submitting the job to is the `gpu` or `multigpu` partition or a private partition that has GPUs available.
+```
+
+Selecting the [partition](https://rc-docs.northeastern.edu/en/latest/hardware/partitions.html#partitions) you want the job to run on the HPC cluster:
+
+```{code-block} matlab
+>> % Specify the partition 
+>> c.AdditionalProperties.Partition = 'partition-name';
+```
+
+Setting the number of cores for the job:
+
+```{code-block} matlab
+>> % Specify cores per node (default: 0)
+>> c.AdditionalProperties.ProcsPerNode = 4;
+```
+
+Setting exclusive use of the node:
+
+```{code-block} matlab
+>> % Set node exclusivity (default: false)
+>> c.AdditionalProperties.RequireExclusiveNode = true;
+```
 
-There are benchmarking scripts and examples available in **`**/shared/centos7/matlab/R2020a/examples/parallel/main/**`** on the cluster.
+```{note}
+The exclusive node option can increase your wait time in the queue so only use if required.
+```
 
-To make the scripts and examples available, add the path to this folder to the list of available paths by doing one of the following:
+If you are running the job on a reservation that has be setup:
 
-- On the MATLAB Home tab, in the **Environment** section, click **Set Path** and add the path to the script.
-- Alternatively, provide the script's full path in the MATLAB command line.
+```{code-block} matlab
+>> % Use reservation 
+>> c.AdditionalProperties.Reservation = 'reservation-name';
+```
 
-The contents of _myParallelAlgorithmFcn_ are as follows:
+Setting the time for the job to run on the HPC cluster:
 
 ```{code-block} matlab
-function [numWorkers,time] = myParallelAlgorithmFcn ()
+>> % Specify the wall time (e.g., 1 day, 5 hours, 30 minutes)
+>> c.AdditionalProperties.WallTime = '1-05:30';
+```
 
-complexities =  [2^18 2^20 2^21 2^22];
-numWorkers = [1 2 4 6 16 32 64];
+```{note}
+You need to make sure the time matches the partition you are submitting to or the jobs will not run.
```

Comment:
You need to make sure the time fits within the max time allowed on the [partition](https://rc.northeastern.edu/partitions/) you are submitting to or the jobs will not run.